### PR TITLE
[enterprise-4.14] OCPBUGS-25695: CP to 4.14

### PR DESCRIPTION
**Pull request by rohennes**
Cherry-pick of #72146 

Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OCPBUGS-25695

Link to docs preview:
https://72431--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performing-platform-verification-latency-tests

### CODE REVIEW

File: modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc [hunk 1]
```diff
@@ -40,7 +40,12 @@ where:
 $ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
 -e DISCOVERY_MODE=true -e FEATURES=performance -e IMAGE_REGISTRY="<disconnected_registry>" \
 -e CNF_TESTS_IMAGE="cnf-tests-rhel8:v{product-version}" \
+<<<<<<< HEAD
 /usr/bin/test-run.sh -ginkgo.focus="\[performance\]\ Latency\ Test"
+=======
+-e LATENCY_TEST_RUNTIME=<time_in_seconds> \
+<disconnected_registry>/cnf-tests-rhel8:v{product-version} /usr/bin/test-run.sh --ginkgo.v --ginkgo.timeout="24h"
+>>>>>>> 8875e10430 (OCPBUGS-25695: Ginkgo test suite now times out in 1hr. Setting flag to 24hr to accomodate latency tests)
```
Comment (swatisehgal) [inline modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc:48]: Please resolve conflict here.

File: modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc [hunk 2]
```diff
@@ -56,8 +61,13 @@ You can run the latency tests using a custom test image and image registry using
 $ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
 -e IMAGE_REGISTRY="<custom_image_registry>" \
 -e CNF_TESTS_IMAGE="<custom_cnf-tests_image>" \
+<<<<<<< HEAD
 -e FEATURES=performance \
 registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} /usr/bin/test-run.sh
+=======
+-e LATENCY_TEST_RUNTIME=<time_in_seconds> \
+registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} /usr/bin/test-run.sh --ginkgo.v --ginkgo.timeout="24h"
+>>>>>>> 8875e10430 (OCPBUGS-25695: Ginkgo test suite now times out in 1hr. Setting flag to 24hr to accomodate latency tests)
```
Comment (swatisehgal) [inline modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc:70]: Same as above.

File: modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc [hunk 3]
```diff
@@ -141,9 +151,9 @@ registry.redhat.io/openshift4/cnf-tests-rhel8:{product-version} \
 +
 [source,terminal,subs="attributes+"]
 ----
-$ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
+$ ppodman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
```
Comment (swatisehgal) [inline modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc:154]:
```suggestion
$ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
```

File: modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc (hunk 1 above)
Comment (rohennes) [reply modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc]: Whoops, thanks for catching, thought I resolved them all.

**Review by swatisehgal — APPROVED** [review]
/lgtm

**Pull request merged**